### ISSUE TITLE
False-positive for 'no-unused-imports' for wildcards with extension functions

### ISSUE THREAD

**Issue by ntr-ross** [pinterest/ktlint#1325]
When allowing wildcard imports and using one to import all extension functions from a package the import is tagged as unused even when an extension function from it is in use in the code.

This is with:
- ktlint 0.43.2
- both Kotlin 1.5.31 and 1.6.10

Attached is a minimal gradle project that exemplifies this.  Just run `ktlint -v` in the project root and you should get:
```
$ ktlint -v
.../app/src/main/kotlin/import_use_bug/App.kt:3:1: Unnecessary import (no-unused-imports)
```
This occurs whether there is just a single extension or multiple extensions defined in the `extensions` package.

TIA.

[import-use-bug.zip](https://github.com/pinterest/ktlint/files/7775076/import-use-bug.zip)

**Comment by paul-dingemans** [comment]
This problem is a duplicate of #1256. The lastest attempt to solve it, in PR https://github.com/pinterest/ktlint/issues/1277, is currently stuck.
